api/hypershift/v1beta1/nodepool_types.go

```diff
@@ -931,6 +931,19 @@ type OpenStackNodePoolPlatform struct {
 	//
 	// +optional
 	ImageName string `json:"imageName,omitempty"`
+
+	// AvailabilityZone is the nova availability zone in which the provider will create the VM.
```

Comment:
Nit
```suggestion
	// availabilityZone is the nova availability zone in which the provider will create the VM.
```

Reply:
yes I wanted to do that but I followed other fields in HCP API...

Reply:
I'll change it anyway :man_shrugging: